include/walker-dropdown.php

```diff
@@ -36,7 +36,7 @@ public function start_el( &$output, $element, $depth = 0, $args = array(), $curr
 		$output .= sprintf(
 			"\t" . '<option value="%1$s"%2$s%3$s>%4$s</option>' . "\n",
 			'url' === $value_type ? esc_url( $element->$value_type ) : esc_attr( $element->$value_type ),
-			! empty( $element->locale ) ? sprintf( ' lang="%s"', esc_attr( $element->locale ) ) : '',
+			! empty( $element->w3c ) ? sprintf( ' lang="%s"', esc_attr( $element->w3c ) ) : '',
```

Comment:
Why did you removed the `locale` and use `w3c` instead? Sorry I might have miss something...

Reply:
Because it's the `w3c` that should be passed normally, it's a bug introduced in an old [PR](https://github.com/polylang/polylang/pull/1039).

Moreover, it was causing a bug in our JS code, as we had WordPress and W3C locales in our HTML that we were comparing, and which were not the same.

Reply:
I should have read the PR description correctly, sorry 🙈